tests/impl.cpp

```diff
@@ -2491,7 +2491,37 @@ result_t test_m_pmulhuw(const SSE2NEONTestImpl &impl, uint32_t iter)
 
 result_t test_mm_prefetch(const SSE2NEONTestImpl &impl, uint32_t iter)
 {
-    return TEST_UNIMPL;
+    struct prefetch_test_t {
+    __m128 a;
+    float r[4];
+  } ;
+    struct prefetch_test_t test_vec[8] = {
```

Comment:
Re-generate the test vectors with simplified values.

Reply:
done